src/progressbar/progressbar.ts

```diff
@@ -0,0 +1,36 @@
+import {Component, Input, ElementRef} from 'angular2/angular2';
+
+@Component({
+  selector: 'ngb-progressbar',
+  template: `
+    <progress class="progress {{extraClasses}}  {{type && 'progress-' + type}}" [value]="value" [max]="max">
+      <div class="progress">
+        <span class="progress-bar" [style.width.%]="calculatePorcentage()"><ng-content></ng-content></span>
+      </div>
+    </progress>
+  `
+})
+export class NgbProgressbar {
+  _value: number;
+  min = 0;
+  extraClasses: string;
+
+  @Input()
+  set value(newValue: number) {
+    this._value = newValue;
+    this.calculatePorcentage();
+  }
+
+  get value() { return this._value; }
+
+  @Input() type: string;
+
+  @Input() max = 100;
+
+  constructor(private element: ElementRef) {
```

Comment:
@Foxandxss could you put this new version in a PR. There are few things that can / should be changed but it is super-hard to comment on the code in a plunk.